### PR TITLE
First generation of Data Catalog

### PR DESCRIPTION
**Pull request by sduskis**

### CODE REVIEW

File: google-cloud-bom/pom.xml [hunk 1]
```diff
@@ -354,6 +354,21 @@
         <artifactId>google-cloud-core-http</artifactId>
         <version>1.69.1-SNAPSHOT</version><!-- {x-version-update:google-cloud-core-http:current} -->
       </dependency>
+      <dependency>
+        <groupId>com.google.cloud</groupId>
+        <artifactId>google-cloud-datacatalog</artifactId>
+        <version>0.87.1-alpha-SNAPSHOT</version><!-- {x-version-update:google-cloud-datacatalog:current} -->
```
Comment (kolea2) [inline google-cloud-bom/pom.xml]: shouldn't the version be 0.0.1-alpha-SNAPSHOT?

File: google-cloud-clients/google-cloud-datacatalog/pom.xml [hunk 2]
```diff
@@ -0,0 +1,65 @@
+<?xml version="1.0"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <artifactId>google-cloud-datacatalog</artifactId>
+  <version>0.87.1-beta-SNAPSHOT</version><!-- {x-version-update:google-cloud-datacatalog:current} -->
```
Comment (kolea2) [inline google-cloud-clients/google-cloud-datacatalog/pom.xml]: also, inconsistency between alpha and beta here

Comment (sduskis) [reply google-cloud-clients/google-cloud-datacatalog/pom.xml]: I'll change to alpha.

File: google-cloud-bom/pom.xml (hunk 1 above)
Comment (sduskis) [reply google-cloud-bom/pom.xml]: We have been using a consistent version across all artifacts that are pre-0.  I'm not sure what we ought to do going forward.

**Pull request closed without merging**